src/main.cpp

```diff
@@ -1136,7 +1140,11 @@ int main(int argc, char *argv[])
                 break;
 
             case '*':
+#ifdef HAVE_AUDIO
                 audioParams.audio_backend = optarg;
+#else
+                std::cerr << "Ignoring --audio-backend. Built without audio support." << std::endl;
```

Comment:
Alternatively, it could be a hard error.

Reply:
Hey, sorry for the late reply. Yeah I think a hard error would be great! Otherwise LGTM.